invenio_app_ils/jwt.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# Invenio-Circulation is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""."""
+
+from __future__ import absolute_import, print_function
+
+from flask_security import current_user
+from invenio_accounts.utils import jwt_create_token
+from invenio_userprofiles.api import current_userprofile
+
+
+def ils_jwt_create_token():
+    """."""
+    user_id = current_user.get_id()
+    additional_data = {"locationPid": 1}
```

Comment:
please add in the issue of users to add the location_pid to the user profile/additional data and then use that one here

Reply:
Done https://github.com/inveniosoftware/invenio-app-ils/issues/71